modules/admin/src/messages/it/admin.php

```diff
@@ -0,0 +1,212 @@
+// added translation in 1.0.0-beta5:
+    'aws_gallery_empty' => 'Seleziona una immagine sulla sinistra per aggiungerla all\'album.',
+    'aws_gallery_images' => 'Album immagini',
+    'layout_useronline_name' => 'Nome',
+    'layout_useronline_mail' => 'E-Mail',
+    'layout_useronline_inactivesince' => 'Non attivo dal',
+// added translations in 1.0.0-beta6
+    'layout_debug_caching' => 'Caching',
+    'debug_state_on' => 'On',
+    'debug_state_off' => 'Off',
+// added translations in 1.0.0-beta7
+    'login_securetoken_mail_subject' => 'LUYA Codice di sicurezza',
+    'login_securetoken_mail' => '<h1>LUYA Codice di sicurezza</h1><p>Utilizza il seguente codice di sicurezza per accedere all\'\area di amministrazione {url}:</p><span style="font-size:22px; letter-spacing:2px; padding:5px 10px; display:inline-block; background-color:#F0F0F0;">{token}</span>',
+    'login_async_token_error' => 'Il codice di sicurezza inserito non è valido.',
+    'login_async_token_globalerror' => 'Errore nel Server, perfavore contatta l\'amministratore di sistema.',
+    'ngrest_crud_csv_export_btn' => 'Genera CSV',
+    'ngrest_crud_csv_export_btn_dl' => 'Download CSV',
+    'layout_filemanager_search_text' => 'Inserisci termine di ricerca...',
+    'layout_filemanager_file_captions' => 'Sottotitolo',
+    'layout_filemanager_file_captions_save_btn' => 'Salva',
+    'mode_user_title' => 'Titolo',
+    'mode_user_firstname' => 'Nome',
+    'mode_user_lastname' => 'Cognome',
+    'mode_user_email' => 'E-Mail',
+    'mode_user_password' => 'Password',
+// added translations in 1.0.0-beta8
+    'aws_tag_new' => 'Aggiungi nuovo elemento',
+    'aws_tag_add' => 'aggiungi',
+    'aws_tag_search' => 'Inserisci termine',
+    'aws_tag_list' => 'Lista di tutti i tag',
+    'dashboard_log_message_add' => 'Aggiungi una nuova riga a <b>{container}</b>.',
+    'dashboard_log_message_edit' => 'Aggiorna una riga esistente in <b>{container}</b>.',
+// rc1
+    'layout_btn_help' => 'Aiuto',
+    'right_panel_support_title' => 'Informazioni &amp; Aiuto',
+    'right_panel_support_tags_title' => 'Tags',
+    'right_panel_support_support_title' => 'Supporto',
+    'right_panel_support_support_text' => 'Se hai qualche domanda, contatta il webmaster o consulta qui <a target="_blank" href="https://luya.io">luya.io</a>.',
+// rc2
+    'ngrest_crud_group_prompt' => 'Raggruppa per campo',
+    'ngrest_crud_filter_prompt' => 'Seleziona Filtro',
+    'ngrest_crud_ajax_search_length' => 'Il termine di ricerca deve essere di almeno 3 caratteri.',
+    'menu_system_logger' => 'Log',
+    'layout_filemanager_detail_replace_file' => 'Sostituisci file',
+    'layout_rightbar_languagelabel' => 'Lingua',
+    'layout_rightbar_savebtn' => 'Salva',
+    'aws_changepassword_notequal' => 'La nuova password e la nuova password ripetuta non sono uguali.',
+    'aws_changeapssword_minchar' => 'La nuova password deve contenere almeno {min} caratteri.',
+    
+// 1.0.0
```

Comment:
Those translations (from 1.0.0 and below) should be translated as well. Thank you very much!